Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/data/SimulationData.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * #%L
+ * This file is part of "Apromore Core".
+ * %%
+ * Copyright (C) 2018 - 2021 Apromore Pty Ltd.
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+package org.apromore.plugin.portal.processdiscoverer.data;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+
+@Builder
+@Getter
+public class SimulationData {
```

Comment:
Could this class be included in the process-simulation-info-Logic module? The reason I'm asking, is the current module dependancy structure. PD-Portal-Plugin depends on process-simulation-info-Logic. If SimulationData stays in PD-Portal-Plugin, then a 'logic' module (process-simulation-info-logic) would depend on a portal plugin module, which would cause a cyclic dependancy and is not very elegant?

Reply:
As this is being merged into development, the alternative would be for me to refactor this into the process-simulation-info-logic module at the time I integrate to consume this

Reply:
Yes, we should move it to Simulation Info Logic, pls do it. I'll wait for Simulation Info Logic to be merged into dev branch and then PD Portal can use it.